misk/src/main/kotlin/misk/web/WebTab.kt

```diff
@@ -26,6 +26,8 @@ class DashboardTab(
   url_path_prefix: String,
   val name: String,
   val category: String = "Container Admin",
+    // TODO(rhall): Deprecate this https://github.com/cashapp/misk/issues/1078
   roles: Set<String> = setOf(),
```

Comment:
are these roles providing any value? couldn't we just show a 403 if one of the backend calls failed with Unauthorized? Is it just to gate showing the Tab? Seems not worth it to me at the expense of maintaining multiple ACL sets.

Reply:
Yeah, that might be simpler.